shims/net/index.ts

```diff
@@ -157,8 +157,8 @@ export class Socket extends EventEmitter {
     this.ws = await new Promise<WebSocket>(resolve => {
       try {
         // ordinary/browser path
-        const wsProtocol = this.useSecureWebSocket ? 'wss:' : 'ws:';
```

Comment:
Can we keep this conditional?

Reply:
Yes, we can allow using `ws:` in the driver, but I'm not sure it's a good idea to support unencrypted ws in the proxy itself. Note that this PR is more for experiments, rather than ready to merge.

Reply:
Agreed, we should support wss: only. But others may want to use ws: (coupled with the experimental JS encryption that's also built into the driver).